README.md

```diff
@@ -0,0 +1,39 @@
+Setting up the environment variables
+------------------------------------
+To setup the Girder instance two users are created: ```esimmonadmin``` and ```esimmon```. The passwords for these users must be set in the ```.env``` file located in ```eSimMon/devops/docker``` under the ```ADMIN_PASSWORD``` and ```ESIMMON_PASSWORD``` keys. Optionally, if a Girder instance has already been built the ```GIRDER_API_KEY``` and ```GIRDER_FOLDER_ID``` can be set in the ```watch.env``` file located in the same directory. If the keys are not set, they will be created automatically when the Girder instance is created.
+
+
+Create and setup the Girder instance
+------------------------------------
+To create the Girder instance, you will need to have set the ```ADMIN_PASSWORD``` and ```ESIMMON_PASSWORD``` keys mentioned above, then run the following command:
+
+```docker-compose -p esimmon -f docker-compose.girder.yml -f docker-compose.ansible.yml up```
+
+This will create both users, as well as an assetstore, collection, default folder, and API key in order to allow the ingest script to populate the database with data from the virtual machine run. This should only need to be run once for the initial setup.
+
```

Comment:
We should say if successful it will end with
```
esimmon_ansible_1 exited with code 0
```
At which point they can use ctrl-c to bring down the stack.